### PR TITLE
chore: throw error when bot password is empty

### PR DESCRIPTION
**Pull request by IvanJobs**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_sprints/taskboard/Compute%20Team/Microsoft%20Teams%20Extensibility/CY23-3.2?workitem=17638955

Throw an error when bot id is valid but bot password is empty.

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -901,6 +901,7 @@
   "driver.botAadApp.log.failExecuteDriver": "Unable to execute action %s. Error message: %s",
   "driver.botAadApp.log.successCreateBotAad": "Created Azure Active Directory application with client id %s.",
   "driver.botAadApp.log.useExistingBotAad": "Used existing Azure Active Directory application with client id %s.",
+  "driver.botAadApp.error.unexpectedEmptyBotPassword": "Unexpected empty bot password. You can choose to add your own generated bot password in env file or clear bot id as well so that a new pair would be generated. action: %s.",
```
Comment (IvanJobs) [inline packages/fx-core/resource/package.nls.json:904]: @sffamily  @MuyangAmigo Hi can you help review this error message in terms of wording? thanks.

Comment (sffamily) [reply packages/fx-core/resource/package.nls.json:904]: Try to make it simple and straightforward:

Bot password is empty. Add it in env file or clear bot id to have bot id/password pair regenerated. action: %s.

**Review by hund030 — DISMISSED** [review]

**Review by yukun-dong — DISMISSED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (blackchoey) [reply packages/fx-core/resource/package.nls.json:904]: Though users can find corresponding envs in `writeToEnvironmentFile` field, do we need to tell users which env is empty / needs to be filled?

**Review by eriolchan — APPROVED** [review]

File: packages/fx-core/resource/package.nls.json (hunk 1 above)
Comment (sffamily) [reply packages/fx-core/resource/package.nls.json]: yes, if it can cause confusion, we need to be very specific, may need to specify the env file name.

**Review by hund030 — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**